plugin/src/main/java/io/jenkins/plugins/casc/Configurator.java

```diff
@@ -176,8 +177,9 @@ default CNode describeStructure(T instance, ConfigurationContext context)
             if(context.getMode().equals("JSONSchema")) {
                 attribute.setJsonSchema(true);
             }
+            CNode cNode = context.lookup(attribute.getType()).describe(instance, context);
```

Comment:
@timja this kind of returns an exception because it cannot be converted into an enumeration, in here: https://github.com/jenkinsci/configuration-as-code-plugin/blob/fd433295bae57aceb9c69b853e9086716299cad0/plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/PrimitiveConfigurator.java#L70

Reply:
If we just print out the configurators for `context.lookup()` we still end up with just the configurator names.